galera/files/my.cnf.bootstrap

```diff
@@ -9,94 +9,200 @@
 {%- from "galera/map.jinja" import slave with context %}
 {%- set service = slave %}
 {%- endif %}
-[MYSQLD]
-user=mysql
-basedir=/usr/
-datadir=/var/lib/mysql
-socket={{ service.socket }}
-pid_file=mysqld.pid
-port=3306
-log_error={{ service.log_file }}
-log_warnings=2
-#log_output=FILE
-### INNODB OPTIONS 
-innodb_buffer_pool_size=9211M
-innodb_flush_log_at_trx_commit=2
-innodb_file_per_table=1
-innodb_data_file_path = ibdata1:100M:autoextend
-## You may want to tune the below depending on number of cores and disk sub
-innodb_read_io_threads=4
-innodb_write_io_threads=4
-innodb_io_capacity=200
-innodb_doublewrite=1
-innodb_log_file_size=1024M
-innodb_log_buffer_size=96M
-innodb_buffer_pool_instances=8
-innodb_log_files_in_group=2
-innodb_thread_concurrency=64
-#innodb_file_format=barracuda
-innodb_flush_method = O_DIRECT
-innodb_autoinc_lock_mode=2
-## avoid statistics update when doing e.g show tables
-innodb_stats_on_metadata=0
-default_storage_engine=innodb
+# MariaDB database server configuration file.
```

Comment:
Maybe you can split mysql config into smaller pieces and use jinja's include to avoid duplicating configuration.